library/common/http/dispatcher.cc

```diff
@@ -477,19 +477,38 @@ const LowerCaseString ClusterHeader{"x-envoy-mobile-cluster"};
 const std::string BaseCluster = "base";
 const std::string BaseWlanCluster = "base_wlan";
 const std::string BaseWwanCluster = "base_wwan";
+const LowerCaseString H2UpstreamHeader{"x-envoy-mobile-upstream-protocol"};
+const std::string H2Suffix = "_h2";
+const std::string BaseClusterH2 = BaseCluster + H2Suffix;
+const std::string BaseWlanClusterH2 = BaseWlanCluster + H2Suffix;
+const std::string BaseWwanClusterH2 = BaseWwanCluster + H2Suffix;
```

Comment:
nit: this would actually be one less line of code if you just used literals ;)

Reply:
looks like this is actually defined twice too?